docs/source/DeveloperGuide/DevelopmentEnvironment/DeveloperInstallation/stuffThatRun.rst

```diff
@@ -31,61 +30,100 @@ you better keep reading.
 
 
 Notes before continuing, on top of what is in section :ref:`editing_code`
----------------------------------------------------------------------------
+=========================================================================
 
-*OS*: a DIRAC server can be installed, as of today, only on SLC6/CC7 OS.
+*OS*: a DIRAC server can be installed, as of today, only on SLC6 (Scientific Linux Cern 6) or CC7 (Cern CentOS 7) OS.
 The reason being some binaries are proved to work only there,
 and this includes several DMS (Data Management) libraries.
 If you have to do many DMS development, you should consider using SLC6 or CC7.
 Or, using a Virtual Machine, or a docker instance. We'll go through this.
 
 
 Stuff you need to have installed, on top of what is in section :ref:`editing_code`
--------------------------------------------------------------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your development machine is a SLC6 or a CC7, probably nothing of what follows.
+
+If your development machine is a Scientific Linux 6 or a CentOS 7, probably nothing of what follows, but I wouldn't be too sure about it.
```

Comment:
No it's not. In the line above there is "C" (which stands for Cern) that is missing in the line below :)